build-tests/sparo-real-repo-test/etc/list-profiles-with-project.txt

```diff
@@ -0,0 +1,21 @@
+Running "sparo list-profiles --project sparo-lib":
+
+[1mSparo accelerator for Git __VERSION__ -[22m[36m https://tiktok.github.io/sparo/[39m
```

Comment:
Including ANSI control characters in `.txt` files is a bit awkward.  For example, the GitHub website doesn't render <kbd>ESC</kbd>.  To a code reviewer, it seems like a mistake.

Similar to __VERSION__, what if we rewrote these codes to a readable ASCII representation, something like:

```
<<ANSI:1m>>Sparo accelerator for Git <<VERSION>> -<<ANSI:22m>><<ANSI:36m>> https://tiktok.github.io/sparo/<<ANSI:39m>>
```

Reply:
Oh wait, apparently I already invented this: 😂

[AnsiEscape.formatForTests()](https://github.com/microsoft/rushstack/blob/661252a40ebca38af8a1868448b9c00e23d7ae6a/libraries/terminal/src/AnsiEscape.ts#L52)